package.json

```diff
@@ -84,6 +85,7 @@
     "redux-thunk": "^2.1.0",
     "reselect": "^2.5.1",
     "sprintf-js": "1.0.3",
+    "type-coerce": "^0.1.0",
```

Comment:
Not sure if I trust this library. Looking at the repo: https://github.com/wacky6/type-coerce/blob/master/lib/type-coerce.js it seems that it doesn't have any stars, there was only one commit, with no issues or PRs... the code is simple enough, but this is rougly the equivalent of copying and pasting from StackOverflow. I'd suggest looking for another library, or implementing it ourselves, or reviewing the `type-coerce` library as if it was part of this PR.